src/web/app/pages-help/instructor-help-page/instructor-help-courses-section/instructor-help-courses-section.component.html

```diff
@@ -65,7 +65,7 @@ <h3 *ngIf="displaySubsection(showQuestion, 0, 5)">Enrolling Students in Courses<
     </div>
   </div>
   <!--Question-->
-  <div class="instr-help-qn3" *ngIf="showQuestion.length === 0 || showQuestion[3]" #question>
+  <div class="instr-help-qn" *ngIf="showQuestion.length === 0 || showQuestion[3]" #question>
```

Comment:
Is this class `instr-help-qn` ever defined? If not used, we can just remove it,

Reply:
Right, it's not used. Removed!